.github/workflows/pr.yml

```diff
@@ -75,11 +75,25 @@ jobs:
           echo "predict result directory missing" >&2 
           exit 1
         fi
-        echo "result lines "$(wc -l example/results_predict/predict_data_AR.csv)
-        if [ "$(cat example/results_predict/predict_data_AR.csv | wc -l)" -lt "2000" ]; then
-          echo "predict result should have at least 2000 lines. But had only $(cat example/results_predict/predict_data_AR.csv | wc -l)" >&2 
+        echo "result lines "$(wc -l example/results_predict/smiles.csv)
+        if [ "$(cat example/results_predict/smiles.csv | wc -l)" -lt "5" ]; then
+          echo "predict result should have at least 5 lines. But had only $(cat example/results_predict/predict_data_AR.csv | wc -l)" >&2 
```

Comment:
Why did the number of lines change that much? Was this wrong before?

Reply:
I did provide another smaller csv to do the preds, but the principle is the same.

Reply:
Cool. Good idea.